libpalaso.yml

```diff
@@ -0,0 +1,128 @@
+# unless you specify --no-cache with dotnet.exe commands or -NoCache with nuget.exe commands. If the package is not in the cache,
+# or the cache isn't used, NuGet then retrieves the package over HTTP .
+#
+# NUGET_PACKAGES        Path to use for the global-packages folder ...          Specified as absolute path.
+
+name: libpalaso
+buildsystem: simple
+build-options:
+    prepend-path: /usr/lib/sdk/mono6/bin
+    append-ld-library-path: /usr/lib/sdk/mono6/lib
+    append-pkg-config-path: /usr/lib/sdk/mono6/lib/pkgconfig
+    env:
+        DOTNET_RUNTIME_ID: org.gnome.Platform
+build-commands:
+    - |
+        set -xueo pipefail
+        pwd
+        ls -FC
+        export NUGET_PACKAGE=`pwd`/packages
+        echo NUGET_PACKAGE=$NUGET_PACKAGE
+        find . -iname nuget.exe -print
+        echo mono build/nuget.exe sources add -name Downloaded -source `pwd`/nuget-sources
+        mono build/nuget.exe sources add -name Downloaded -source `pwd`/nuget-sources
+        mono build/nuget.exe sources disable -name nuget.org
+        mono build/nuget.exe sources list
+        mono build/nuget.exe restore -PackagesDirectory `pwd`/packages -force -verbosity detailed
+        msbuild /t:Compile /p:Configuration=Release /p:RestorePackagesPath=`pwd`/packages build/Palaso.proj
+        # copy where these can be found by the BloomDesktop nuget package code
+        install -d /app/libpalaso
+        install output/Release/net461/SIL.Core.dll /app/libpalaso
+        install output/Release/net461/SIL.Core.pdb /app/libpalaso
+        install output/Release/net461/SIL.Core.Desktop.dll /app/libpalaso
+        install output/Release/net461/SIL.Core.Desktop.pdb /app/libpalaso
+        install output/Release/net461/SIL.Media.dll /app/libpalaso
+        install output/Release/net461/SIL.Media.pdb /app/libpalaso
+        install output/Release/net461/SIL.Windows.Forms.dll /app/libpalaso
+        install output/Release/net461/SIL.Windows.Forms.pdb /app/libpalaso
+        install output/Release/net461/SIL.Windows.Forms.GeckoBrowserAdapter.dll /app/libpalaso
+        install output/Release/net461/SIL.Windows.Forms.GeckoBrowserAdapter.pdb /app/libpalaso
+        install output/Release/net461/SIL.Windows.Forms.Keyboarding.dll /app/libpalaso
+        install output/Release/net461/SIL.Windows.Forms.Keyboarding.pdb /app/libpalaso
+        install output/Release/net461/SIL.Windows.Forms.WritingSystems.dll /app/libpalaso
+        install output/Release/net461/SIL.Windows.Forms.WritingSystems.pdb /app/libpalaso
+        install output/Release/net461/SIL.WritingSystems.dll /app/libpalaso
+        install output/Release/net461/SIL.WritingSystems.pdb /app/libpalaso
+sources:
+    - type: git
+      url: https://github.com/sillsdev/libpalaso.git
+      branch: Bloom-5.2
+      commit: c4fd324c4024640ba6652e5184d66bf7962672d2
+      disable-shallow-clone: true
+    - type: file
+      url: https://dist.nuget.org/win-x86-commandline/v6.2.1/nuget.exe
```

Comment:
so this is x86 only?
(we don't support x86)

Reply:
nuget.exe is a C# assembly that runs just fine on a x64 Linux box using mono.  I suspect it was built for AnyCPU despite the win-x86-commandline in the path.

Reply:
ah ok.